OmgvaPOS/src/ScheduleManagement/Controller/ScheduleController.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OmgvaPOS.ScheduleManagement.DTOs;
+using OmgvaPOS.ScheduleManagement.Models;
+using OmgvaPOS.ScheduleManagement.Service;
+
+namespace OmgvaPOS.ScheduleManagement.Controller
+{
+    [ApiController]
+    [Route("schedules")]
+    public class ScheduleController(IScheduleService scheduleService) : Microsoft.AspNetCore.Mvc.Controller
+    {
+        private readonly IScheduleService _scheduleService = scheduleService;
+
+        [HttpPost]
+        [Authorize(Roles = "Admin,Owner,Employee")]
+        [ProducesResponseType<EmployeeSchedule>(StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public IActionResult CreateEmployeeSchedule([FromBody] CreateEmployeeScheduleRequest request)
+        {
+            var schedule = _scheduleService.CreateEmployeeSchedule(request);
```

Comment:
Shouldn't there be checking for ability to manage business?

Reply:
Same in other endpoints

Reply:
I'm not sure of how I can aquire business id to compare it with the one from the token.